src/main/java/net/dv8tion/jda/core/entities/Message.java

```diff
@@ -1254,4 +1262,247 @@ public boolean isImage()
             return height > 0 && width > 0;
         }
     }
+
+    /**
+     * Represents a {@link net.dv8tion.jda.core.entities.Message} activity like Spotify.
+     */
+    class Activity
```

Comment:
Don't be specific about spotify, this is used by a lot more with RPC

Reply:
I did change that